src/vfocus.js

```diff
@@ -315,3 +315,8 @@ VFocus.prototype.indexOf = function(focus){
 VFocus.prototype.spliceChildren = function(){
   return Array.prototype.splice.apply(this.children(), arguments);
 };
+
+VFocus.prototype.prependChildren = function(children){
```

Comment:
Could we move these under a:
```
/**
* Children
*/
```

Or whichever name makes sense. They'r now in the "Iteration methods" group. I think we should modularise this at some point, but for now.